build/azure-pipelines/sql-product-build.yml

```diff
@@ -7,8 +7,7 @@ resources:
 jobs:
 - job: Compile
   pool:
-    vmImage: 'Ubuntu-16.04'
-  container: linux-x64
```

Comment:
we need this container.

Reply:
does this mean, we will use this container on our own agent?